app/javascript/components/layout/CustomTable.js

```diff
@@ -324,6 +333,7 @@ class CustomTable extends React.Component {
 }
 
 CustomTable.propTypes = {
+  dataType: PropTypes.string.required,
   columnData: PropTypes.array,
   rowData: PropTypes.array,
   totalRows: PropTypes.number,
```

Comment:
Not sure I understand enough about the JS side of this to know why this is happening. Is there a way to resolve this? @timwongj 

![Screen Shot 2021-04-21 at 12 35 01 PM](https://user-images.githubusercontent.com/24628687/115603930-ff328c00-a29d-11eb-8205-9b471a4329e5.png)

Reply:
This error happens for me on a patient show page

Reply:
great catch, fixed here: https://github.com/SaraAlert/SaraAlert/pull/806/commits/79c99d1bdff2281ff7c2e159842e8936ae6c9ec3

thanks @jkufro